### PR TITLE
Upgrade to centralised NodeJS codegen

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: sdk/nodejs/README.md [hunk 1]
```diff
@@ -1,18 +1,3 @@
...
-This package is typically used in conjunction with the [Pulumi Azure provider](https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/azure/).
-
-If this is your first time using this package, these two resources may be helpful:
-
-* [Azure Getting Started Guide](https://www.pulumi.com/docs/quickstart/azure): Get up and running quickly.
-* [Azure Pulumi Setup Documentation](https://www.pulumi.com/docs/quickstart/azure/configure/): How to configure Pulumi
-    for use with your Azure account.
-
-Use the navigation below to see detailed documentation, including sample code, for each of the supported Azure services.
-
```
Comment (pgavlin) [inline sdk/nodejs/README.md:15]: We shouldn't use this. Did you delete the original `README.md` when generating these changes?

Comment (pgavlin) [reply sdk/nodejs/README.md:15]: (same feedback for the other provider PRs)

Comment (stack72) [reply sdk/nodejs/README.md:15]: yes, I deleted the entire tree - does this need to be manually managed? If so, should it be in all parts of our SDK?

Comment (pgavlin) [reply sdk/nodejs/README.md:15]: The root-level `README.md` does need to be manually managed, yes, but the nested readmes are generated.

Comment (lukehoban) [reply sdk/nodejs/README.md:15]: I believe it's actually intentional that the nested READMEs are being deleted.  Per https://github.com/pulumi/home/issues/734#issuecomment-606323575.

**Pull request merged**